arrows/klv/klv_read_write.txx

```diff
@@ -343,6 +366,57 @@ klv_ber_oid_length( T value )
   return ( _int_bit_length( value ) + 6 ) / 7;
 }
 
+// ---------------------------------------------------------------------------
+template < class Iterator >
+double
+klv_read_float( Iterator& data, size_t length )
+{
+  static_assert( std::numeric_limits< float  >::is_iec559 &&
+                 std::numeric_limits< double >::is_iec559,
+                 "platform not IEEE-754 compliant" );
```

Comment:
```suggestion
                 "platform is not IEEE-754 compliant" );
```
...or maybe "non-IEEE-754 platform is not supported", but either is fine, and really, even missing the "is" isn't a big deal.